angular/js/controllers/submitController.js

```diff
@@ -45,6 +45,7 @@ angular.module('nuswhispersApp.controllers')
     $scope.submitConfession = function () {
         $scope.form.submitButtonDisabled = true;
         $scope.confessionData.categories = $scope.form.selectedCategoryIDs;
+        $scope.confessionData.token = localStorageService.get('confessionToken');
```

Comment:
Hehe it can even be a hardcoded gibberish string that has no meaning:

`$scope.confessionData.KZ7Ng9N9VaNgHLye = localStorageService.get('YVgQXdna7g4CDAkD')`.

People wouldn't be able to figure out unless they read the back end source code 😂